src/cookbooks/bigquery/access.md

```diff
@@ -101,22 +113,30 @@ pip install google-cloud-bigquery
 python -c 'from google.cloud import bigquery; print([d.dataset_id for d in bigquery.Client().list_datasets()])'
 ```
 
-## From Spark
+### Spark
+
+> This method requires [BigQuery Access](#BigQuery-Access-Request) to be provisioned.
+
 We recommend the [Storage API Connector](https://github.com/GoogleCloudPlatform/spark-bigquery-connector) for accessing
 BigQuery tables in Spark as it is the most modern and actively developed connector. It works well with the BigQuery
 client library which is useful if you need to run arbitrary SQL queries (see example Databricks notebook) and load their
 results into Spark.
 
-### On Databricks
-The `shared_serverless_python3` cluster is configured with shared default GCP credentials that are automatically picked
-up by BigQuery client libraries. It also has the Storage API Connector library added as seen in the example
-[Python notebook](https://dbc-caf9527b-e073.cloud.databricks.com/#notebook/141939).
+### Databricks
+
+[Databricks Notebooks](https://docs.databricks.com/notebooks/index.html) provide an interactive
+computational environment, similar to Jupyter. If you are a Mozilla employee, you should be able to access it via [`sso.mozilla.com`](https://sso.mozilla.com).
 
-### On Dataproc
-Dataproc is Google's managed Spark cluster service. Accessing BigQuery from there will be faster than from Databricks
-because it does not involve cross-cloud data transfers.
+The `shared_serverless_python3` cluster is configured with shared default GCP credentials that are automatically picked up by BigQuery client libraries. It also has the Storage API Connector library added as seen in the example [Python notebook](https://dbc-caf9527b-e073.cloud.databricks.com/#notebook/141939).
+
+### Dataproc
```

Comment:
Maintain relative heading level:
```suggestion
#### Dataproc
```

Reply:
This makes sense, but it should be moved up (to before) Databricks. I'll move this up in a seperate commit, will have to adjust some prose.

Reply:
Wound up not doing this, see above.